### ISSUE TITLE
Issue with another mod, Late Game Upgrades (and other mods that buff the shovel)

### ISSUE THREAD

**Issue by riktoi** [x753/Lethal-Company-Mimics#14]
As the mod checks the force of the attack (which is 1 in vanilla), any mod that increases the damage the shovel does will cause the mimic to aggro in 2 hits, or even 1 hit. 

I suggest making it increase anger by 1 no matter the force to make the mimic consistently aggro in 3 hits.

**Comment by x753** [comment]
I didn't consider this, I'll see if I can make some shovel-specific adjustments in a future update.

**Comment by ShadokuSan** [comment]
Yeah, this scared the heck outta me a couple of times before I realized what was happening. Though if possible (or not too inconvenient for you to do), it'd be nice to have an option to keep this behavior at some point down the line. Hitting doors is a fairly easy method to check for mimics, and if you're using any mods that up your damage then a lot of the normal threats are not an issue - it'd be a trade-off of sorts; trading the ability to easily detect a mimic to deal with monsters easier.
